enterprise/README.md

```diff
@@ -39,20 +24,34 @@ This will give you Flyway Command-line's usage instructions.
 
 To do anything useful however, you must pass the arguments that you need to the image. For example:
 
-`docker run --rm redgate/flyway -licenseKey="FL01..." -url=jdbc:h2:mem:test -user=sa info`
+`docker run --rm redgate/flyway -licenseKey="FL01..." -url=sqlite:/flyway/project/dev.db info`
```

Comment:
this url needs `jdbc:` at the start and should be in the same format as other url uses

Reply:
Yes it should !

Reply:
Reverted it anyway - needs TLC later